datageneration/generate_combination_table.py

```diff
@@ -16,11 +17,15 @@
 
 class QueryCombinationGenerator(object):
     def __init__(self, geolocation_file: str, tag_combinations: List[TagCombination],
-                 property_examples: List[TagPropertyExample], max_distance_digits: int):
-        self.entity_tag_combinations = list(filter(lambda x: 'core' in x['comb_type'], tag_combinations))
-        self.area_generator = AreaGenerator(geolocation_file)
+                 property_examples: List[TagPropertyExample], max_distance_digits: int,
+                 percentage_of_two_word_areas: float, prop_generating_contain_rel: float,
+                 ratio_within_radius_within: float):
+        self.entity_tag_combinations = list(filter(lambda x: 'core' in x.comb_type.value, tag_combinations))
+        self.area_generator = AreaGenerator(geolocation_file, percentage_of_two_word_areas)
         self.property_generator = PropertyGenerator(property_examples)
-        self.relation_generator = RelationGenerator(max_distance_digits=max_distance_digits)
+        self.relation_generator = RelationGenerator(max_distance_digits=max_distance_digits,
+                                                    prop_generating_contain_rel=prop_generating_contain_rel,
+                                                    ratio_within_radius_within=ratio_within_radius_within)
 
     def index_to_descriptors(self, index):
```

Comment:
I noticed that we don't use this func anymore. We can remove it from the main script.